app/routes/index.js

```diff
@@ -16,9 +12,9 @@ router.post('/login', async function(req, res, next) {
         const error = "wrong secret key";
         res.render('login', {title: 'Login', data: error});
     }else{
-        const token = await generateToken({secret_key: req.body.secret_key, error: ""});
-        req.cookie('token', token);
-        res.json({ token });
+        const token = await tokenUtil.generateToken({secret_key: req.body.secret_key});
```

Comment:
What does this section do?

Reply:
This is the authentication function, we take the user info, and we don't want to directly save that in the cookie. Therefore, we need to make sure we encode/hash it using token, so that it is safe.